### PR TITLE
Draft of Getting Started Docs

### PR DESCRIPTION
**Pull request by MaggieFero**
Here's my first thwack at documentation for how to get started with the Vuln Cost code extension, as described in Issue #50 . I had some trouble getting it to prompt me to Authorize again, so I'm especially interested in having somebody who just got started review this PR, because I think that part might be wrong, but I remember the options being confusing at first.

I fully expect this to need revisions before merge, but I've had that issue on hold for a long time now, so I want to get at least something in here.

### CODE REVIEW

File: GETTING-STARTED.MD [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Authorizing Your Account
+The first time you use the Vuln Cost Security Scanner on a new device, you will need to authorize it by logging into your Snyk account before seeing remediation advice. 
+1. First, find a package with known vulnerabilities in your `package.json` or equivalent. 
+2. Click in the name of that package, then click the blue extension icon that appears in the left margin.
+A dropdown menu opens.
+3. Select **Fix Vulns** from the dropdown.
+VSCode will ask if you want to open the External Link. You must open the external link to continue. 
+![Open The External Website - Vuln Cost](https://user-images.githubusercontent.com/1719499/113234978-2bd32500-9257-11eb-9d21-70a02abcfa1b.png)
+4. Click Open. 
+An **Authenticate for CLI** prompt appears. This is because the Snyk CLI tool performs the underlying lookups to determine whether your packages are known vulnerable. ![Authenticate for CLI - Vuln Cost](https://user-images.githubusercontent.com/1719499/113234976-2b3a8e80-9257-11eb-8eda-ba02a7e40569.png)
```
Comment (bmvermeer) [inline GETTING-STARTED.MD]: This is not true the CLI tool does not perform the underlying lookups.
We just reused the CLI authentication flow.

File: GETTING-STARTED.MD [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Adding the Extension
+To get started with the Vuln Cost Security Scanner extension for Visual Studio Code (VSCode), you must first install the *Vuln Cost - Security Scanner* by `snyk-security.vscode-vuln-cost` from the **Extensions Marketplace**. You may need to restart VSCode to see warnings.
```
Comment (bmvermeer) [inline GETTING-STARTED.MD:2]: Great call out! In some cases, people restarted VSCode multiple times before it worked. (Currently we don't know why) If there are any problems make sure VS Code is fully shut down and restarted.

File: GETTING-STARTED.MD [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+# Viewing Vulnerability Details
+Click within the name of a dependency that lists a number of vulns, and a blue extension icon will appear in the left margin of VSCode. When you click the extension icon, a dropdown appears with two options: **Fix Vulns** or **Learn about this package**.  
+
+If you select **Fix Vulns**,  VSCode asks if you would like to go to the external webpage. If you haven’t authorized Snyk CLI since installing the extension, you will need to [authorize your account](#authorizing-your-account) to actually see fix options. If you have already authorized Snyk CLI, the **Output** panel lists direct and indirect vulnerabilities known to affect your package version, as well as a version to upgrade to for possible remediation.
```
Comment (bmvermeer) [inline GETTING-STARTED.MD]: authorize your existing account, or sign up for a free account if you don't have one yet.

**Review by bmvermeer — CHANGES_REQUESTED** [review]
Great stuff, just some minor changes

File: GETTING-STARTED.MD (hunk 1 above)
Comment (MaggieFero) [reply GETTING-STARTED.MD]: Ahhhhhh, okay. From the end-user perspective, it was CLI auth and appears in the Snyk web UI as a CLI client, so it seemed like this was just a different UX for the CLI. Thanks for that clarification!

**Comment by MaggieFero** [comment]
Thanks for the review! I've completed the requested changes, but I'm not sure how to submit the new version for review/merge. Do you already see the new version here now, or should I do something to update the files submitted for this PR (or does it require a new PR altogether? hopefully not anything that drastic)?

**Comment by Ryuno-Ki** [comment]
> Do you already see the new version here now

We do.

**Review by bmvermeer — APPROVED** [review]

**Pull request merged**